scripts/appleseedMaya/renderGlobals.py

```diff
@@ -1,11 +1,10 @@
-
 #
 # This source file is part of appleseed.
 # Visit https://appleseedhq.net/ for additional information and resources.
 #
 # This software is released under the MIT license.
 #
-# Copyright (c) 2016-2019 Esteban Tovagliari, The appleseedhq Organization
+# Copyright (c) 2016-2019 Esteban Tovagliari, Luis Barrancos, The appleseedhq Organization
```

Comment:
We only mention the initial author in copyright notices, all other authors are included as `The appleseedhq Organization`.